theme_bookstore/views/snippets/s_shape_image.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_shape_image" inherit_id="website.s_shape_image">
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Book collection
+    </xpath>
+    <!-- Paragraph -->
+    <xpath expr="//p" position="replace" mode="inner">
+        Explore our extensive collection of books across various genres. Whether you’re looking for bestsellers, classics, or hidden gems, we offer a curated selection to satisfy every reader's curiosity and passion.
```

Comment:
Just my opinion here... but for a litterature-oriented business, I'd avoid using `you're` for `you are`. To me, this is a "written" text, not the transcription of a "spoken" text. (But I'm not a native English speaker - so I might be totally wrong)